### ISSUE TITLE
New repositories do not seem to have a base64 encoded node_id

### ISSUE THREAD

**Issue by jdbaldry** [magit/forge#417]
I expected that `forge-add-repository` should be able to add new repositories.
However, `forge-add-repository` errors trying to base64 decode the node_id of a new repository.
It seems like the node_id format has changed?

To reproduce:
```elisp
(debug-on-entry 'forge-add-repository)
(let ((owner "jdbaldry")
      (repo "test-github-node-ids"))
  ;; requires admin privileges.
  ;; (ghub-delete (format "/repos/%s/%s" owner repo) nil)
  (ghub-post "/user/repos" nil :payload `((name . ,repo)))
  (message "id=%s" (ghub-repository-id owner repo))
  (forge-add-repository (format "git@github.com:%s/%s.git" owner repo)))
```

Backtrace:
```elisp
Debugger entered--Lisp error: (error "Invalid base64 data")
  (base64-decode-string "R_kgDOGI8rzw")
  (#f(compiled-function (class host owner name &optional stub) "Return (OUR-ID . THEIR-ID) of the specified repository.\nIf optional STUB is non-nil, then the IDs are not guaranteed to\nbe unique.  Otherwise this method has to make an API request to\nretrieve THEIR-ID, the repository's ID on the forge.  In that\ncase OUR-ID derives from THEIR-ID and is unique across all\nforges and hosts." #<bytecode 0x1092ccd>) forge-github-repository "github.com" "jdbaldry" "test-github-node-ids" nil)
  (apply #f(compiled-function (class host owner name &optional stub) "Return (OUR-ID . THEIR-ID) of the specified repository.\nIf optional STUB is non-nil, then the IDs are not guaranteed to\nbe unique.  Otherwise this method has to make an API request to\nretrieve THEIR-ID, the repository's ID on the forge.  In that\ncase OUR-ID derives from THEIR-ID and is unique across all\nforges and hosts." #<bytecode 0x1092ccd>) forge-github-repository ("github.com" "jdbaldry" "test-github-node-ids" nil))
  (forge--repository-ids forge-github-repository "github.com" "jdbaldry" "test-github-node-ids" nil)
  (#f(compiled-function (&rest rest) "((host owner name) &optional remote demand)\n\nReturn the repository identified by HOST, OWNER and NAME." #<bytecode 0x113b7c1>) ("github.com" "jdbaldry" "test-github-node-ids") nil create)
  (apply #f(compiled-function (&rest rest) "((host owner name) &optional remote demand)\n\nReturn the repository identified by HOST, OWNER and NAME." #<bytecode 0x113b7c1>) ("github.com" "jdbaldry" "test-github-node-ids") (nil create))
  (forge-get-repository ("github.com" "jdbaldry" "test-github-node-ids") nil create)
  (#f(compiled-function (url &optional remote demand) "Return the repository at URL." #<bytecode 0x113b755>) "git@github.com:jdbaldry/test-github-node-ids.git" nil create)
  (apply #f(compiled-function (url &optional remote demand) "Return the repository at URL." #<bytecode 0x113b755>) "git@github.com:jdbaldry/test-github-node-ids.git" (nil create))
  (forge-get-repository "git@github.com:jdbaldry/test-github-node-ids.git" nil create)
  (#f(compiled-function (url) "Add a repository to the database.\nOffer to either pull topics (now and in the future) or to only\npull individual topics when the user invokes `forge-pull-topic'." (interactive #f(compiled-function () #<bytecode 0x2bebd51>)) #<bytecode 0x22a44d9>) "git@github.com:jdbaldry/test-github-node-ids.git")
  (apply #f(compiled-function (url) "Add a repository to the database.\nOffer to either pull topics (now and in the future) or to only\npull individual topics when the user invokes `forge-pull-topic'." (interactive #f(compiled-function () #<bytecode 0x2bd4161>)) #<bytecode 0x22a44d9>) "git@github.com:jdbaldry/test-github-node-ids.git")
* (forge-add-repository "git@github.com:jdbaldry/test-github-node-ids.git")
  (let ((owner "jdbaldry") (repo "test-github-node-ids")) (ghub-post "/user/repos" nil :payload (list (cons 'name repo))) (ghub-repository-id owner repo) (forge-add-repository (format "git@github.com:%s/%s.git" owner repo)))
  (eval (let ((owner "jdbaldry") (repo "test-github-node-ids")) (ghub-post "/user/repos" nil :payload (list (cons 'name repo))) (ghub-repository-id owner repo) (forge-add-repository (format "git@github.com:%s/%s.git" owner repo))) nil)
  (elisp--eval-last-sexp nil)
  (eval-last-sexp nil)
  (funcall-interactively eval-last-sexp nil)
  (call-interactively eval-last-sexp nil nil)
  (command-execute eval-last-sexp)
```

**Comment by tarsius** [comment]
Yes. I have already addressed that in 483baae56df8b1d8f1a9136512d23e106c831c0b.

**Comment by jdbaldry** [comment]
Sorry I missed that commit. Thanks for the fix!
